contracts/UFragmentsPolicy.sol

```diff
@@ -37,16 +38,27 @@ contract UFragmentsPolicy is Ownable {
         uint256 timestampSec
     );
 
+    event LogRebaseV2(
+        uint256 indexed epoch,
+        uint256 exchangeRate,
+        uint256 targetRate,
+        int256 requestedSupplyAdjustment,
+        uint256 timestampSec
```

Comment:
In theory, the timestamp is available in the block header, but I can see how it might be easier for downstream consumers to have this natively in the log stream with the other data.

Is subgraph able to index block timestamps?

Reply:
yea we can get rid of the timestamp.